frontend/src/AddLogoComponent/AddDraggableComponent.jsx

```
@@ -189,24 +197,51 @@ export default function AddDraggableComponent({ id,title, onRemove, onMouseMove,
+         height="20px" 
+         viewBox="0 -960 960 960" 
+         width="20px" 
+         fill={editLogoBoxColor}>
+          <path 
+          d="m256-200-56-56 224-224-224-224 56-56 
+          224 224 224-224 56 56-224 
+          224 224 224-56 56-224-224-224 
+          224Z"/>
+          </svg>
+          </span>
+      </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)